step-capstone/src/components/AddItemButton.js

```diff
@@ -16,14 +16,14 @@ export default function AddItemButton(props) {
     };
 
     const open = Boolean(anchorEl);
-
     return (
         <div>
             <Fab aria-label="add" onClick={handleClick}>
                 <AddIcon />
             </Fab>
             <FormPopover
                 data={undefined}
+                startDate={props.startDate}
```

Comment:
Why do we need to pass in a startDate into the form?

Reply:
I pass the startDate to initialize the startDate of any new TravelObject. Before, we'll initialize the date as the current date (new Date(), now they are props.startDate). You can see the use of this in Add[TravelObject] files.